internal/helper/aws-sso.fish

```diff
@@ -19,6 +19,18 @@ function aws-sso-profile
   end
 end
 
+function __aws_sso_profile_complete
+  set --local _args (string split -- ' ' $AWS_SSO_HELPER_ARGS)
+  set -q AWS_SSO_HELPER_ARGS; or set --local _args -L error --no-config-check
+  set -l cur (commandline -t)
+
+  set -l cmd "aws-sso list $_args --csv -P Profile=$cur Profile"
```

Comment:
should `aws-sso` be `{{ .Executable }}` ?

Reply:
Updated